packages/tools/pie-icons-vue/icons/IconVoucherFilledLarge.js

```diff
@@ -0,0 +1,34 @@
+import _mergeJSXProps from '@vue/babel-helper-vue-jsx-merge-props';
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { iconSize, updateContextData } from './configs-vue';
+
+export default {
+    name: 'IconVoucherFilledLarge',
+    props: {
+        iconSize: iconSize.large,
+    },
+    functional: true,
+    render (h, ctx) {
+        ctx.data = updateContextData(ctx, 'c-pieIcon c-pieIcon--voucherFilledLarge');
+        return h('svg', _mergeJSXProps([{
+            attrs: {
+                xmlns: 'http://www.w3.org/2000/svg',
+                viewBox: '0 0 32 32',
+                role: 'presentation',
+                focusable: 'false',
+            },
+        }, ctx.data]), [h('path', {
+            attrs: {
+                fill: '#242E30',
+                'fill-rule': 'evenodd',
```

Comment:
Just curious, since you're merging jsx props, wouldn't it take camelCase variables like `fillRule`?

Reply:
If you'd like to start a wider discussion with the team about refactoring these generated files then I'd be glad to join in